src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -3,550 +3,292 @@
-            values[(int)KnownColor.RosyBrown] = unchecked((int)0xFFBC8F8F);
-            values[(int)KnownColor.RoyalBlue] = unchecked((int)0xFF4169E1);
-            values[(int)KnownColor.SaddleBrown] = unchecked((int)0xFF8B4513);
-            values[(int)KnownColor.Salmon] = unchecked((int)0xFFFA8072);
-            values[(int)KnownColor.SandyBrown] = unchecked((int)0xFFF4A460);
-            values[(int)KnownColor.SeaGreen] = unchecked((int)0xFF2E8B57);
-            values[(int)KnownColor.SeaShell] = unchecked((int)0xFFFFF5EE);
-            values[(int)KnownColor.Sienna] = unchecked((int)0xFFA0522D);
-            values[(int)KnownColor.Silver] = unchecked((int)0xFFC0C0C0);
-            values[(int)KnownColor.SkyBlue] = unchecked((int)0xFF87CEEB);
-            values[(int)KnownColor.SlateBlue] = unchecked((int)0xFF6A5ACD);
-            values[(int)KnownColor.SlateGray] = unchecked((int)0xFF708090);
-            values[(int)KnownColor.Snow] = unchecked((int)0xFFFFFAFA);
-            values[(int)KnownColor.SpringGreen] = unchecked((int)0xFF00FF7F);
-            values[(int)KnownColor.SteelBlue] = unchecked((int)0xFF4682B4);
-            values[(int)KnownColor.Tan] = unchecked((int)0xFFD2B48C);
-            values[(int)KnownColor.Teal] = unchecked((int)0xFF008080);
-            values[(int)KnownColor.Thistle] = unchecked((int)0xFFD8BFD8);
-            values[(int)KnownColor.Tomato] = unchecked((int)0xFFFF6347);
-            values[(int)KnownColor.Turquoise] = unchecked((int)0xFF40E0D0);
-            values[(int)KnownColor.Violet] = unchecked((int)0xFFEE82EE);
-            values[(int)KnownColor.Wheat] = unchecked((int)0xFFF5DEB3);
-            values[(int)KnownColor.White] = unchecked((int)0xFFFFFFFF);
-            values[(int)KnownColor.WhiteSmoke] = unchecked((int)0xFFF5F5F5);
-            values[(int)KnownColor.Yellow] = unchecked((int)0xFFFFFF00);
-            values[(int)KnownColor.YellowGreen] = unchecked((int)0xFF9ACD32);
-            s_colorTable = values;
-        }
+            // The appended group of SystemColors (i.e. not sequential with WindowText above)
+            (byte)Interop.User32.Win32SystemColors.ButtonFace,
+            (byte)Interop.User32.Win32SystemColors.ButtonHighlight,
+            (byte)Interop.User32.Win32SystemColors.ButtonShadow,
+            (byte)Interop.User32.Win32SystemColors.GradientActiveCaption,
+            (byte)Interop.User32.Win32SystemColors.GradientInactiveCaption,
+            (byte)Interop.User32.Win32SystemColors.MenuBar,
+            (byte)Interop.User32.Win32SystemColors.MenuHighlight
+        };
 
-        private static void EnsureColorNameTable()
-        {
-            // no need to lock... worse case is a double create of the table...
-            //
-            if (s_colorNameTable == null)
-            {
-                InitColorNameTable();
-            }
-        }
+        public static uint GetSystemColorArgb(KnownColor color)
```

Comment:
I must be missing something about what's compiled where.  This isn't used on Unix?

Reply:
Oh, I see, there's an ifdef earlier for FEATURE_WINDOWS_SYSTEM_COLORS. Ok.